src/main/java/sirius/biz/process/ProcessContext.java

```diff
@@ -73,6 +73,13 @@
      */
     void addDebugTiming(String counter, long millis);
 
+    /**
+     * Increents the given performance counter by one.
+     *
+     * @param counter the counter to increment
+     */
+    void addCounter(String counter);
```

Comment:
i think it may be worthwhile to rename this to `incrementCounter` as a method addCounter especially with no int parameter rather implies that this creates/attaches something new to the process instead of incrementing a counter by 1.

Reply:
right, this would also look like a lazy guy just lovelessly copy+pasted the method above :)